pkg/config/config_template.yaml

```diff
@@ -646,6 +646,13 @@ api_key:
   #
   # logs_no_ssl: false
 
+  ## @param expected_tags_duration - int - optional - default: 15
```

Comment:
more generally, I'm not sure we should document this option in the config_template, since it's a troubleshooting option most users shouldn't need to set. WDYT?